package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "my-todo-list-app",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet). This project uses webpack to bundle js modules. User can add, remove, edit, list items. Built with JavaScript, HTML & CSS",
+  "private": true,
+  "scripts": {
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d dist",
+    "test": "jest --coverage --watch --verbose",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "build": "webpack --config webpack.config.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/steveWDamesJr/ToDo-list-application.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/steveWDamesJr/ToDo-list-application/issues"
+  },
+  "homepage": "https://steveWDamesJr.github.io/ToDo-list-application",
```

Comment:
- [ ] I recommend changing the name `todo-list` to the `leaderboard` in this field.